### PR TITLE
Fix typo / grammatical error

### PR DESCRIPTION
**Pull request by jinalskothari**
Just 2 tiny fixes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -98,7 +98,7 @@ Show only relevant data when loading via server. Fold your content to save bytes
 
 
 ##### Page Caching
-Well now you can cache a page in SSR. Pretty simple. Just add cache option to route.  
+We'll now you can cache a page in SSR. Pretty simple. Just add cache option to route.  
 `{ cache: { enable: true, duration: 10000}}`
```
Comment (yash-thakur) [inline README.md]: Here it will come "Well, now you can cache..."

Comment (jinalskothari) [reply README.md]: Ah, right!
Would you prefer a new PR or should I push an additional commit?

Comment (yash-thakur) [reply README.md]: Whatever you feel fine.

**Comment by jinalskothari** [comment]
Closing this. Please see https://github.com/Atyantik/react-pwa/pull/84

**Pull request closed without merging**